tutorial/tutorial.txt

```diff
@@ -1,133 +0,0 @@
-#######################
-## Dr. Colin Davenport
-## Wochenende pipeline: bacterial metagenome commands. April 2021
-## Intended to find bacteria
-######################
-
-#################
-# Start: Input fastq files as *R1_fastq must be in same folder. 
-# This assumes you have a SLURM cluster to run the pipeline on
-################
-
-
-
-# Segment 1, preparation, rename, run pipeline
-
-#0 Setup 
-Set up Wochenende following the commands on the Wochenende github page. Download a reference sequence and edit config.yaml
-# Now copy get_wochenende.sh to your current directory
-cp (your_wochenende_path)/get_wochenende.sh .
-
-#1 edit the location "path_we" in the following script, then run it to collect the Wochenende files and directories needed.
-nano get_wochenende.sh 
-# Now run the script to set the needed scripts and directories for the pipeline
-bash get_wochenende.sh
-
-
-#2. Wochenende needs sample names like sputum_R1.fastq and sputum_R2.fastq to run correctly. So rename your files.
-srun gunzip *.fastq.gz
-rename 's/_001//g' *.fastq
```

Comment:
should we add this information in the Wiki?

Reply:
yep, done, good point